pkg/util/proc/process_unix.go

```diff
@@ -135,6 +135,9 @@ func findProcessSearch(binary, cmdlinePattern string) (*UnixProcess, error) {
 func processes() ([]*UnixProcess, error) {
 	d, err := os.Open("/proc")
 	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, nil
```

Comment:
if `/proc` isn't found seems like your linux system would have other issues. What if we just log a warning?